sonic-net/sonic-buildimage
# [Marvell] platform renaming for innovium

#### Why I did it
Renaming of platforms and asic types for marvell
This PR has dependencies on the below submodule PRs

<html><body>

Repo | PR title | State
-- | -- | --
sonic-swss |[ [Marvell] platform renaming for innovium](https://github.com/sonic-net/sonic-swss/pull/3252) | Open
sonic-utilities |[ platform renaming for innovium](https://github.com/sonic-net/sonic-utilities/pull/3474) | Open
sonic-sairedis |[ [Marvell] platform renaming for innovium](https://github.com/sonic-net/sonic-sairedis/pull/1408) | Open
sonic-mgmt |[Changes to support chip name change from innovium to marvell-teralynx](https://github.com/sonic-net/sonic-mgmt/pull/14330) | Open

</body>
</html>




#### How I did it
renamed platform folder names and asic types as below
platform/innovium renamed to platform/marvell-teralynx


#### How to verify
Verify marvell-teralynx  builds going through, and verify the images in teralynx switches and verified linkup of interfaces

##### Work item tracking
- Microsoft ADO **(number only)**:

#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)
master

#### Description for the changelog
NA

#### Link to config_db schema for YANG module changes
NA

#### A picture of a cute animal (not mandatory but encouraged)
